test/dns/query.cpp

```diff
@@ -389,26 +389,26 @@ TEST_CASE("The libevent resolver works as expected") {
     });
 
     reactor->run_with_initial_event([=]() {
```

Comment:
Changes in this file are because a host that we used to test has changed IPv6 address.